### ISSUE TITLE
Clarify Spring Boot versions supported by Sentry

### ISSUE THREAD

**Issue by mhalbritter** [spring-io/start.spring.io#1647]
I can't find anything on [their website](https://docs.sentry.io/platforms/java/guides/spring-boot/), so I've asked in [their Discord](https://discord.com/channels/621778831602221064/621778831602221072/1309430041653940286).

Right now, we have marked it as compatible with 3.2 only.

**Comment by verglor** [comment]
Currently it states "Requires Spring Boot >= 3.3.0 and < 3.3.0-M1." which is quite confusing... 🤔

**Comment by mhalbritter** [comment]
Yeah. Right now, there's no supported Boot version which let's you use it. 

The sentry team opened https://github.com/getsentry/sentry-java/issues/3931 and they will reach back to us when they've done their testing.

**Comment by mhalbritter** [comment]
Superseded by https://github.com/spring-io/start.spring.io/pull/1713.
